app/graphql/input_objects/department_attributes.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module InputObjects
+  class DepartmentAttributes < Types::BaseInputObject
+    description 'Attributes for modifying a department entry'
+    argument :name, String, 'The name of the department', required: true
+    argument :location, String, 'The location of the deparment', required: true
```

Comment:
'The location of the deparment' -> 'The location of the department'

Reply:
Done